frontend/src/components/dashboard/DashboardHeader.tsx

```diff
@@ -7,28 +7,18 @@ import { Button, Col, FormControl, InputGroup, Row } from "react-bootstrap";
 import AccountsApi from "../../libraries/explorer-wamp/accounts";
 import BlocksApi from "../../libraries/explorer-wamp/blocks";
 import TransactionsApi from "../../libraries/explorer-wamp/transactions";
-import DetailsApi, * as D from "../../libraries/explorer-wamp/details";
+import { SubConsumer } from "../../context/SubscriptionProvider";
```

Comment:
Please, don't create shortcut names that are hard to read. It makes the review and maintenance process harder.